src/utils.py

```diff
@@ -1,10 +1,13 @@
-import logging
 import os
 import boto3
 import logging
 
-logger = logging.getLogger()
-logger.setLevel(logging.DEBUG)
+log_level = os.getenv('LOG_LEVEL', logging.ERROR)
+logger = logging.getLogger(__name__)
+logger.setLevel(log_level)
+
+my_lambda = boto3.client('lambda', os.getenv('AWS_DEPLOYMENT_REGION', 'us-south-10'))
```

Comment:
Again, not sure we actually want to default to `us-south-10` if `AWS_DEPLOYMENT_REGION` isn't provided.

Reply:
Fixed.

Reply:
I don't know how to get github to show updated comments.  But at this point, the function is configured with a cloudWatch event that names the error handler error alarm.  Then the lambda function receive alarms for the error handler error alarm and toggles on the alarm state (alarm/ok).  I think you were looking at files while I was switch from a cron job to event driven.